test/unit/specs/components/SearchPage.spec.js

```diff
@@ -152,5 +167,14 @@ describe('SearchPage.vue', () => {
       state.selectedSystems = [];
       expect(wrapper.vm.selectedSystemIds).toEqual([]);
     });
+
+    test('selectedSystemLabels', () => {
+      const selectedSystem = state.selectedSystems[0];
+      const wrapper = shallowMount(SearchPage, { store, localVue });
+      const foundSystem = bodySystems.find(x => { return x.id === selectedSystem.id; });
```

Comment:
Since you're returning the value of a single expression, this arrow function can be simplified:

`bodySystems.find(x => x.id === selectedSystem.id);`